score-http/score-http/src/main/java/org/oagi/score/repo/BusinessInformationEntityRepository.java

```diff
@@ -649,6 +653,16 @@ public SelectBieListArguments setOwnedByDeveloper(Boolean ownedByDeveloper) {
             }
             return this;
         }
+        
+        public SelectBieListArguments setTenantBusinessCtx(boolean isAdmin, List<ULong> userTenantIds ) {
```

Comment:
Code Style: Remove the space (' ') between the parameter name and the closing brace.